src/components/addpeer/WindowsTab.tsx

```diff
@@ -3,16 +3,18 @@ import React, {useState} from 'react';
 import { Button } from "antd";
 import TabSteps from "./TabSteps";
 import { StepCommand } from "./types"
+import {getConfig} from "../../config";
+const {latestVersion} = getConfig();
 
 export const WindowsTab = () => {
 
-    const releaseVersion = '0.6.2'
+    const releaseVersion = latestVersion ? latestVersion : "v0.6.3"
     const [steps, setSteps] = useState([
         {
             key: 1,
             title: 'Download and run Windows installer:',
             commands: (
-                <Button type="primary" href={`https://github.com/netbirdio/netbird/releases/download/v${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`} target="_blank">Download NetBird</Button>
+                <Button type="primary" href={`https://github.com/netbirdio/netbird/releases/download/${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`} target="_blank">Download NetBird</Button>
```

Comment:
Codacy has a fix for the issue: [Replace `········<Button·type="primary"·href={`https://github.com/netbirdio/netbird/releases/download/${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`}·target="_blank">Download·NetBird` with `<Button⏎··········type="primary"⏎··········href={`https://github.com/netbirdio/netbird/releases/download/${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`}⏎··········target="_blank"⏎········>⏎··········Download·NetBird⏎········`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
        <Button
```